deployment/helm/ditto/templates/nginx-ingress.yaml

```diff
@@ -0,0 +1,843 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+{{- if .Values.ingress.enabled -}}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "{{ .Values.ingress.namespace }}"
+  labels:
+    name: "{{ .Values.ingress.namespace }}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-node-health-check-conf
+  namespace: "{{ .Values.ingress.namespace }}"
+data:
+  nginx-node-health-check.conf: |
+    # config for health check container running together with the ingress controller
+    worker_processes auto; #auto-detect number of logical CPU cores
```

Comment:
If this nginx is only used for healh-check, I would limit the `worker_processes` to `1` instead of using `auto` which will use the amount of available CPU cores of the worker node it runs on as value.

Reply:
it is health check and there is no need for more then 1 worker process

Reply:
done